src/command/private_key_template.rs

```diff
@@ -49,7 +50,8 @@ pub fn put_sign<const R: usize, T: trussed::Client>(
     let key_id = match attr {
         SignatureAlgorithm::EcDsaP256 => put_ec(ctx.lend(), CurveAlgo::EcDsaP256)?,
         SignatureAlgorithm::Ed255 => put_ec(ctx.lend(), CurveAlgo::Ed255)?,
-        SignatureAlgorithm::Rsa2k | SignatureAlgorithm::Rsa4k => {
+        SignatureAlgorithm::Rsa2k => put_rsa(ctx.lend(), Mechanism::Rsa2kPkcs)?,
+        SignatureAlgorithm::Rsa4k => {
             warn!("Key import for RSA not supported");
```

Comment:
This general statement sounds wrong because it does work for 2048 bit. Shouldn't this be more specific and refer to 4096 bit RSA?